OtripleS.Web.Api/Brokers/Storage/StorageBroker.Students.cs

```diff
@@ -1,10 +1,13 @@
 ﻿using Microsoft.EntityFrameworkCore;
 using OtripleS.Web.Api.Models.Students;
+using System;
 
 namespace OtripleS.Web.Api.Brokers.Storage
 {
     public partial class StorageBroker
     {
         public DbSet<Student> Students { get; set; }
+
+        public Student GetStudent(Guid StudentId) => Students.Find(StudentId);
```

Comment:
There's a couple of things here that you could do:
1. Brokers speak the database language from a naming convention perspective, so instead of `GetStudent` we should say `SelectStudent` - in your case here it would be `SelectStudentById`.
2. Use the Asynchronous way of doing things wherever you can, in this case here you could do:
```csharp
public ValueTask<Student> SelectStudentByIdAsync(Guid studentId) => this.Students.FindAsync(studentId);
```
3. Input parameters in any method are camelCased - so `studentId` instead of `StudentId`.

Reply:
Great feedback thanks!

1. I wasn't sure on this, have not worked with brokers in the past. This makes sense, and I'll remember it.

2. I meant to do this when I seen the assignment, but evidently did not once I sat down to write the code.

3. Also meant to do this, my bad.

These should all be fixed by commit 5ca6e44